docs/README.md

```diff
@@ -1,101 +1,104 @@
-4. `cd docs` to enter the docs folder
-5. Run `npm install`
-   Note: Run this from the project's `docs` folder, not the root of the repository on your machine!
-6. Run `npm run dev` to launch a preview
-7. Visit `localhost:3000/` to see a live preview of the docs
+1. [Fork the Nimbus GitHub project](https://github.com/nimbus-zone/nimbus/fork) into your own account.
+2. Clone your fork to your local machine. See [GitHub's instructions for more information](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#cloning-your-forked-repository).
+3. From your terminal, `cd` to the directory you cloned into (`cd nimbus` by default).
+4. Run `pnpm i` or `ni` to install the project dependencies.
+5. Run `pnpm --filter nimbus-docs dev` or `nr --filter nimbus-docs dev` to run the development server.
+
+Once the server has started up, you can access the live preview at http://localhost:3000/. This preview reloads when you save changes to the documentation.
 
 ### Contributing
 
-When you are ready to submit work back to the main Nimbus repo, create a PR.
-
-1. If it has been a bit, synchronize your fork with the upstream repo on GitHub.
-2. Do your work in a branch on your fork
-   Use `git checkout -b branchNameToUse` to create a working branch separate from `main`.
-3. Do your work in your preferred editor
-4. Commit changes often and write meaningful commits
-5. Push the changes from your local machine to your fork on GitHub
-6. Go to your fork of the Nimbus project in your GitHub account
-7. Select the **Pull Request** tab
-8. Select **New Pull Request**
-9.  Confirm the repo/branches to compare
-   - Base repo should be **elk-zone/elk**
-   - base branch should be **main**
-   - Head repository should be your fork
-   - Compare branch should be your working branch you want to submit
-   If you don't see four drop-downs, be sure you are comparing across forks.
-10. Add a description of the changes your request makes
-11. Select **Add Pull Request**
+When you are ready to submit work back to the main Nimbus repo, create a pull request (PR).
+
+1. If your fork has gone out of sync with the main repository, synchronize your fork with the upstream repo on GitHub. See [GitHub's instructions for more information](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
+2. Do your work in a branch on your fork.
+
+      Run `git checkout -b branchNameToUse` in your terminal to create a working branch. Replace `branceNameToUse` with a meaningful branch name.
+3. Do your work in your preferred editor.
+4. Commit changes often and write meaningful commits.
+5. Push the changes from your local machine to your fork on GitHub.
+6. Go to your fork of the Nimbus project in your GitHub account.
+7. Select the **Pull Request** tab.
+8. Select **New Pull Request**.
+9.  Confirm the repo/branches to compare.
+   - The base repo should be **nimbus-zone/nimbus**.
```

Comment:
can you replace `nimbus-zone` with `nimbus-town`? the repo has been renamed

Reply:
@userquin Done